distro/configuration/ampathforms/ct_service_enrolment_v1.0.json

```diff
@@ -826,7 +826,7 @@
                   }
                 }
               ],
-              "id": "patientIdentifier"
+              "id": "CurrentArtRegimen"
```

Comment:
It is by convention that "variable names" are camelCase and start with a small letter?